htdocs/accountancy/journal/bankjournal.php

```diff
@@ -158,6 +158,7 @@
 if ($in_bookkeeping == 'notyet') {
 	$sql .= " AND (b.rowid NOT IN (SELECT fk_doc FROM ".MAIN_DB_PREFIX."accounting_bookkeeping as ab  WHERE ab.doc_type='bank') )";
 }
+$sql .= " GROUP BY b.rowid";
```

Comment:
Be careful, if you set sql_mode=ONLY_FULL_GROUP_BY, you have a sql error because some fileds are not in "GROUP BY"
Maybe try to add all fields needed in "GROUP BY" or make the group using PHP code and not in SQL request.